src/com/xmartlabs/slackbot/CommandManager.kt

```diff
@@ -153,8 +153,23 @@ object CommandManager {
             """
                Hey, thanks for sharing your feedback! :muscle:
                If you want to propose a new feature you can open a <https://github.com/xmartlabs/slackbot/discussions | GitHub Discussion> or an <https://github.com/xmartlabs/slackbot/issues | issue>.
-               
-               All contributions are welcomed! :github: 
+
+               All contributions are welcomed! :github:
+            """.trimIndent()
+        },
+        Command(
+            "frogs",
+            title = "Frogs information :frog:",
+            description = "Give me more information about Frogs :pray:"
+        ) { _, _ ->
+            """
+               Hey, here there is useful information about Frogs:
+               * If you have no idea what is Frogs and how it works, check this <https://www.notion.so/xmartlabs/Frogs-2668aa7ab5d64f0d99f16d4233e6d5fa | link>.
+               * We have new website! Check it out at https://frogs.xmartlabs.com/
```

Comment:
Maybe in a link format? What do you think?
```suggestion
               * We have new website! Check it out at <https://frogs.xmartlabs.com/ | frogs.xmartlabs.com>
```

Reply:
sure!